mlflow/projects/databricks.py

```diff
@@ -218,13 +208,11 @@ def get_run_result_state(self, databricks_run_id):
     def jobs_runs_cancel(self, databricks_run_id):
         response = self._databricks_api_request(
             endpoint="/api/2.0/jobs/runs/cancel", method="POST", json={"run_id": databricks_run_id})
-        _check_response_status_code(response)
```

Comment:
No need to check this since we perform the necessary check in `http_request_safe` (called from self._databricks_api_request)